script/sendmail.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+#for saving the mail temporarily
+random=$RANDOM
+
+#Mail related functionality
+mail=$(which mailutils)
```

Comment:
`mailutils` may not be available on all platform. e.g. CentOS 7 has no such command or package.

Reply:
That is why mailutils is checked to see if it is installed.

Reply:
On other platforms, the package providing `mail` is not `mailutils`. On Debian and derivatives, it seems the name. Here, checking `mail` may be enough.